pkg/interface/src/views/components/MentionText.tsx

```diff
@@ -36,57 +31,49 @@ export function MentionText(props: MentionTextProps) {
 }
 
 export function Mention(props: {
-  ship: string;
   contact: Contact;
   contacts?: Contacts;
   group: Group;
+  scrollWindow?: HTMLElement;
+  ship: string;
 }) {
-  const { contacts, ship } = props;
+  const { contacts, ship, scrollWindow } = props;
   let { contact } = props;
-
-  contact = (contact?.color) ? contact : contacts?.[ship];
-
+  contact = contact?.color ? contact : contacts?.[ship];
+  const history = useHistory();
   const showNickname = useShowNickname(contact);
-
   const name = showNickname ? contact?.nickname : cite(ship);
-  const [showOverlay, setShowOverlay] = useState(false);
-  const onDismiss = useCallback(() => {
-    setShowOverlay(false);
-  }, [setShowOverlay]);
-  const onClick = useCallback(() => {
-    setShowOverlay(true);
-  }, [setShowOverlay]);
-
   const group = props.group ?? { hidden: true };
+  const [showOverlay, setShowOverlay] = useState(false);
 
-  const history = useHistory();
+  const toggleOverlay = () => {
+    setShowOverlay((value) => !value);
+  };
```

Comment:
wrapping this in `useCallback` might be nice, just to dodge unnecessary rerenders

Reply:
Thanks, will address